docs/modules.rst

```diff
@@ -5,4 +5,3 @@ Modules
     :glob:
 
     modules/cfme/cfme
-    modules/fixtures/fixtures
```

Comment:
modules/cfme/cfme picks up cfme/fixtures, and the base fixtures directory no longer exits.